src/DependencyInjection/Configuration.php

```diff
@@ -112,8 +112,8 @@ private function addResettingSection(ArrayNodeDefinition $node): void
                             ->addDefaultsIfNotSet()
                             ->children()
                                 ->scalarNode('template')->cannotBeEmpty()->defaultValue('@SonataUser/Admin/Security/Resetting/email.html.twig')->end()
-                                ->scalarNode('address')->end()
-                                ->scalarNode('sender_name')->end()
+                                ->scalarNode('address')->cannotBeEmpty()->end()
+                                ->scalarNode('sender_name')->cannotBeEmpty()->end()
```

Comment:
This was better without the cannotBeEmpty no ? This way it's really optional.
What do you plan to do @Hanmac

Reply:
i removed the required on the top of resetting,
so when you just don't configure it, it should not require the mailer

the main goal for this MR, outside from making it not require the mailer if not need to,
is for a very fresh symfony project to just `composer require sonata-project/user-bundle` without having it crash in the `cache:clear` because of the missing sonata_user.yml having missing the resetting configuration

Reply:
Yes, and I agree with this change.
That's why IMHO we shouldn't revert this.

The goal of this PR is to remove the `cannotBeEmpty`, so I was surprised you added it back in the last commit.

Reply:
Cannot be empty does not make the field required, it just make it when you define it, you cannot define it empty.

You are talking about isRequired.

Reply:
Oh I see ; my bad.